src/modules/site-v2/templates/tools/pairwise_indel_finder/submit.html

```diff
@@ -192,11 +176,21 @@ <h1 class="modal-title fs-5" id="generate-primers-modal-label">Thank You</h1>
 // Query the database for indel data based on the currently selected species, strains, & chromosome
 // TODO: Do we have to worry about multiple queries running at once? Will we need to invalidate an older one?
 function run_query() {
+
+  // Get form data as an object
+  var data = form_data_to_object('form-submit');
+
+  // Update start and stop to be full chromosome size
+  data['start'] = 1;
+  data['stop']  = current_chrom_size;
+
+  // Run query
   query_running = true;
   $.ajax({
```

Comment:
side note: we can standardize network calls using the build-in`fetch()`. Eg: 
```
fetch(url, { method: "POST", body: JSON.stringify{body}, headers: { 'Content-Type': 'application/json'} )
```

Reply:
By this, do you mean replace the AJAX calls with fetch?  Happy to add that to my backlog if so.

Reply:
Yes, please -- this is part of the master-plan to remove dependencies on the jQuery lib.